### ISSUE TITLE
TSLint ignores indent rule

### ISSUE THREAD

**Issue by ffxsam** [palantir/tslint#4097]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 3.0.1
- __Running TSLint via__: CLI and VS Code

#### TypeScript code being linted

```ts
const fakeObj = {
  test: 23,
  works: false,
     badlyIndented: true,
};
```

with `tslint.json` configuration:

```json
{
  "$schema":
    "https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/tslint.json",
  "defaultSeverity": "warning",
  "extends": ["tslint:recommended"],
  "linterOptions": {
    "exclude": ["node_modules/**"]
  },
  "rules": {
    "arrow-parens": false,
    "curly": [true, "ignore-same-line"],
    "forin": false,
    "indent": [true, "spaces", 2],
    "interface-name": false,
    "max-line-length": false,
    "no-consecutive-blank-lines": false,
    "no-console": false,
    "no-empty": false,
    "object-literal-key-quotes": [true, "as-needed"],
    "object-literal-sort-keys": false,
    "only-arrow-functions": false,
    "ordered-imports": false,
    "quotemark": [true, "single", "avoid-escape"],
    "radix": false,
    "trailing-comma": [
      true,
      {
        "multiline": {
          "objects": "always",
          "arrays": "always",
          "functions": "never",
          "typeLiterals": "ignore"
        }
      }
    ]
  }
}
```

#### Actual behavior
Bad indentation is ignored

#### Expected behavior
TSLint should complain about bad indentation

**Comment by adidahiya** [comment]
duplicate of #2814
